### DISCUSSION TITLE
foobar2000 ABX 盲听测试指南

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#24] [Announcements]
# foobar2000 ABX 盲听测试指南

如何科学地评判两个近似的音频是否存在人类可感的差异？毫无疑问，盲听是个非常好的方法。

https://zexwoo.blog/posts/tutorials/abx-test/

**Comment by Lifeeeeeeee** [comment]
把一首24bit 48khz flac封装的歌转成MP3 320，确实没听出啥区别测试失败
出于好奇看了下网易云下载的“无损音质”为flac封装的16bit 44.1k，mp3都听不出来了，惶论这个水平
所以是否可以直接说明流媒体的音源水平已经完全可以满足需要
（不过网易云的wasapi输出好像是shared mode来着，其他流媒体平台不太清楚，那么如果有朝一日加入独占模式岂不直接齐活？

**Comment by uamind** [comment]
我把24bit 48khz flac封裝的歌转成MP3 320，确实没听出啥区别测试失败

**Comment by uamind** [comment]
hifi的提升就是從耳機的提升來的，耳機和推力，綫材和碼率封裝是真的騙局，只能説音源區別不大。我前端zx300搭配n3，就是3.5換到4.4口明顯的提升就是推力的提升，所謂平衡接口主要就是推力的提升。hifi確實玄學。綫材更換是一點提升都沒，音源還可以有點。所謂的老燒都是群騙自己的。說什麽換綫能有提升還有音源區別巨大，根本就是假的，什麽都沒有換一副好的耳機提升來的實在

**Comment by Lapis-Git** [comment]
foo_abx 2.1 report
foobar2000 v2.1.2
2024-02-14 14:17:10

File A: realized seraph - void.flac
SHA1: 7e482807cef6254eb76a9ce2188ab6b4228d8f4e
Gain adjustment: -9.04 dB
File B: realized seraph.mp3
SHA1: 7c8159a86f58e0717df158c153c3853bd5bf52ce
Gain adjustment: -9.04 dB

Output:
默认 : 主声音驱动程序
Crossfading: NO

14:17:10 : Test started.
14:20:02 : 01/01
14:21:50 : 01/02
14:22:47 : 02/03
14:23:18 : 03/04
14:23:39 : 04/05
14:24:39 : 05/06
14:26:04 : 06/07
14:26:30 : 07/08
14:26:57 : 08/09
14:28:44 : 09/10
14:28:57 : 10/11
14:29:21 : 11/12
14:29:38 : 12/13
14:29:55 : 12/14
14:30:02 : 12/15
14:30:11 : 13/16
14:30:11 : Test finished.

 ---------- 
Total: 13/16
p-value: 0.0106 (1.06%)

 -- signature -- 
25aadfcc5c1d85eefdd43c28f118c2297283510f

把24bit，48kHz flac转成mp3 320kbps，好像成功了
截了一小段对比着听，倒是能听出来一点差别
要是把两个音源单独拉出来听我绝对分不出来
